### ISSUE TITLE
Reduce unused CSS  

### ISSUE THREAD

**Issue by sumitshinde-84** [FlowFuse/website#2305]
### Description

Reduce unused rules from stylesheets and defer CSS not used for above-the-fold content to decrease bytes consumed by network activity 

here is the link to of the resource : - https://developer.chrome.com/docs/lighthouse/performance/unused-css-rules/?utm_source=lighthouse&utm_medium=lr

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

**Comment by sumitshinde-84** [comment]
cc @joepavitt

**Comment by cstns** [comment]
Did you run a lighthouse report?

**Comment by sumitshinde-84** [comment]
Yes, I have used Lighthouse and this platform as well : https://pagespeed.web.dev/analysis/https-flowfuse-com/pfb6c687ry?form_factor=mobile
